bin/lint-migrations-file/src/change/strict.clj

```diff
@@ -1,12 +1,13 @@
 (ns change.strict
```

Comment:
the unstrict versions are for old migrations, now we moved all of them to `000_legacy_migrations.yaml` we don't need to worry about it anymore